### DISCUSSION TITLE
Laptop with EOS v8.0 not booting today

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#593] [Q&A]
So, 

As a supporter, I have access to v8.0, which I installed on my Dell XPS 13 9333 laptop that I've had for literally a decade. 

All was well. Yesterday I wasn't paying attention and the battery for dangerously low before I shut the lid. 

In previous versions of EOS (v7.1), typically it would require a restart after charging after this scenario. 

This proceeded very much the same way, only after seeing the EOS logo and the spinner, the screen goes dark never to return. 

If I touch the power button I get the logo and spinner again for a moment and then the machine turns off.

I'm hopefully I don't have to reinstall, that's a pain. Any idea on what happened and how to recover? 

Thanks

**Comment by alainm23** [comment]
Could you try logging in from safe mode?

**Comment by doobes** [comment]
How?

**Comment by doobes** [comment]
Ok,

Doing a system summary, I'm told that the LVM Physical Volumes: not ok (BAD).

What now?

**Comment by doobes** [comment]
![PXL_20241011_131900896](https://github.com/user-attachments/assets/ab10457a-3c9c-4018-829b-82187b4474e8)
![PXL_20241011_131746095](https://github.com/user-attachments/assets/a72d6eeb-490c-481a-8557-a2eec8348e3b)

Screenshots

**Comment by doobes** [comment]
Suggestions?

**Comment by doobes** [comment]
I've run through this:

https://ostechnix.com/how-to-fix-broken-ubuntu-os-without-reinstalling-it/

To no avail.

I have network access and terminal access.

Is there a way to fix EOS without a reinstall?

**HELP!!!!!**

**Comment by doobes** [comment]
I'm at the command line via CTRL-ALT-F1. Logged in as the normal user. Both internal and NAS shares mounted.  Connected to the network. Everything functional except Wayland.

Doing loginctl results in:

```
SESSION  UID USER  SEAT  TTY  STATE  IDLE SINCE
      2 1000 chris seat0 tty1 active no   -

1 sessions listed.
```

Doing loginctl session-status 2 results in:


```
2 - chris (1000)
  Since: Fri 2024-10-11 09:56:32 EDT; 9min ago
  State: active
 Leader: 1054 (login)
   Seat: seat0; vc1
    TTY: tty1
 Remote: no
Service: login
   Type: tty
  Class: user
   Idle: no
   Unit: session-2.scope
         ├─1054 /bin/login -p --
         ├─1644 -bash
         └─2451 loginctl session-status 2

Oct 11 09:58:55 XPS13 gnome-session-binary[2112]: WARNING: Falling back to non-systemd startup procedure due to error: GDBus.Error:org.freedesktop.systemd1.NoSuchUnit: Unit gnome-session-tty@gnome.target not found.
Oct 11 09:58:56 XPS13 gnome-session[2112]: gnome-session-binary[2112]: WARNING: Could not get session id for session. Check that logind is properly installed and pam_systemd is getting used at login.
Oct 11 09:58:56 XPS13 gnome-session-binary[2112]: WARNING: Could not get session id for session. Check that logind is properly installed and pam_systemd is getting used at login.
Oct 11 09:58:56 XPS13 gnome-session-binary[2112]: Entering running state
Oct 11 10:00:19 XPS13 gnome-session[2394]: gnome-session-binary[2394]: WARNING: Falling back to non-systemd startup procedure due to error: GDBus.Error:org.freedesktop.systemd1.NoSuchUnit: Unit gnome-session-tty@gnome-wayland.target not found.
Oct 11 10:00:19 XPS13 gnome-session-binary[2394]: WARNING: Falling back to non-systemd startup procedure due to error: GDBus.Error:org.freedesktop.systemd1.NoSuchUnit: Unit gnome-session-tty@gnome-wayland.target not found.
Oct 11 10:00:19 XPS13 gnome-session[2394]: gnome-session-binary[2394]: WARNING: Could not get session id for session. Check that logind is properly installed and pam_systemd is getting used at login.
Oct 11 10:00:19 XPS13 gnome-session-binary[2394]: WARNING: Could not get session id for session. Check that logind is properly installed and pam_systemd is getting used at login.
Oct 11 10:00:19 XPS13 gnome-session-binary[2394]: Entering running state
Oct 11 10:00:19 XPS13 gnome-session-f[2403]: Cannot open display:
```

How do I repair this?

**Comment by doobes** [comment]
Well,

Not much help going on here.

Thusfar, I've tried:
```
sudo dpkg --configure -a
sudo apt install --reinstall libpam-systemd
```
with no change.

I've tried: 

`sudo apt install --reinstall elementary-desktop`

I've tried:
```
sudo apt purge elementary-desktop
sudo apt autoremove
sudo apt install elementary desktop
```
All with no change.

Does **ANYONE** have any suggestions?

**Comment by jeremypw** [comment]
To be honest, if there could be data corruption it could be quicker and safer in the long run to reinstall.  OS8 is still an unstable early release version under development so can break.  You should keep all important data on a separate partition or disc.  I have had to reinstall several times for various reasons and getting to a base system from a live USB only takes 10 minutes or so.  Granted you have to install apps and configure things but I have the saved the necessary commands ready to paste into a terminal to do most of it.

You could try out OS8 risk-free in a virtual machine like VirtualBox or Gnome Boxes although there will be limitations on what you an do.

If you do install better download the latest ISO to do it.

Ideally the OS should safely shutdown when the battery reaches a critical level - if it did not then an issue needs raising ...

**Comment by doobes** [comment]
Chances are I'm going to reinstall.  But, like you, I'm concerned about why this happened. So, I'm not going to reinstall until Danielle and friends have had a lookie see.

All my data is backed up.  I'm not losing any data.  It's the time to set stuff up that was lost. Even well documented it takes me about a day to do an install.

I keep a detailed install log in a Trilium instance that I've set up on my network.

**Comment by jeremypw** [comment]
Fair enough.  The Early Access version is best used for testing the default out-of-the -box stuff with minimal customization.

**Comment by doobes** [comment]
Absolutely, 

But if issues that people have run into while testing aren't going to be responded to (or even acknowledged), then why bother posting the issues?

**Reply by teamcons** [reply]
If you want to report an issue, the best approach is to make sure you can reproduce it (else, how are others going to check it ?),
and narrow it down to the best of your abilities (which you do here, its amazing to have this much log)
check on the repository for the component you have an issue with, if it is already reported
if not, create a new Issue Report in the Issue tab

The DE is a community effort, there are a lot of moving parts outside of elementary's scope. Nobody is responsible or paid for monitoring various channels and giving a nice sounding answer that gives a good vibe.
Its frustrating. I know. I do my part. Please dont be angry at people for doing what they can do, thats really demotivating and shoos away anyone who wanna help.

**Reply by doobes** [reply]
I get it. It's complex.

Maybe there isn't an answer. Who knows. At this point it's moot as I've done a fresh install of the latest. It's not without it issues, but so far so good. 

It just would have been nice to have the request acknowledged. 

It's very off-putting to see other, fairly trivial, requests responded to and resolved and not being at least acknowledged. 

If no one going to bother to respond, why should I bother to report issues?

I posted on the discord thing, and to be honest, that's a cluster.

**Comment by vjr** [comment]
its only a rag tag community - like i said - be patient - someone might eventually be able to help.

**Comment by doobes** [comment]
Well, 

I'm not only looking for help, I'm looking to see if anyone bothered to look into this issue, particularly as I don't seem to be the only one experiencing it. 

Kind of disappointing if you ask me, particularly after seeing other topics responded to.

I get it, these folks are volunteers, but so am I. And my time is valuable too.

If it was a one off issue, that's fine, but who knows at this juncture?

I know enough about Linux to be dangerous at this point. I'm not looking to become an expert, only to have a DE that I can rely on.

**Reply by teamcons** [reply]
I have no idea how to troubleshoot your issue

but issues with power, suspend/recover are usually the Linux Kernel, or another underlying part, not the DE
and looking at the logs it looks more like an issue with SystemD somewhere.
Unfortunately Kernel and SystemD arent really in the hands of Danielle and gang

If you want to check whether the issue is systematic, you'll need to reproduce the issue, and inspect the logs, but this i dont know.

**Comment by jeremypw** [comment]
Sadly, as a an App/UI developer I do not have the knowledge to dig deep into booting issues :disappointed: .  Booting can fail for a number of reasons so reported issues may not have the same cause and may depend on details of the user's setup making them challenging to investigate I imagine.  In this particular case, it appears the install was booting OK but failed after the battery was inadvertently left to run down.  I looked into this a bit and notice that there is no UI to specify what should happen in this instance. The Gnome setting related to this seems to have been removed for some reason. I see that (on OS7.1 at least) the file `/etc/Upower/upower.conf` controls this and defaults to the action "HybridSleep" with fallbacks.  Maybe safer to increase the critical battery percentage and/or change the default action to "PowerOff"?  I know my lap top still consumes significant power when suspended although when I had OS8 installed on bare metal the battery did run out overnight while suspended several times without causing (additional) boot problems.  I always had to boot it via recovery mode anyway.  Do I understand correctly that you were not even getting the recovery mode menu?

**Comment by doobes** [comment]
I can get to recovery mode.  This "seems" to be a fairly limited instance.  I've purposely not done much here in the hope (which is now fading) of not having to reinstall.

I can get to a terminal with all my mapped drives connected. I've gone through a couple of things as recommended on the innerwebs, reinstall the desktop, etc all with no effect. 

Again, I'm going to leave this machine as is for now.  I'll wait for an official release before doing anything further.

Appreciate your feedback.

**Reply by teamcons** [reply]
reinstalling the desktop is just reinstalling the facade. The issue here is the building is sitting on broken pillars. 

Id try some "sudo apt install --reinstall" on random systemctl packages, hoping the issue lies there, but from experience, whenever it is kernel or systemD issue i spare myself a neverending headache and just go nuclear, because theres never a promise some obscure thingie wont break at a random time later in the same general direction
Like here maybe systemctl breaks because something earlier breaks.
